package.json

```diff
@@ -4,7 +4,9 @@
   "description": "The Devfest 2017 website! Brought to you by https://adicu.com.",
   "license": "MIT",
   "scripts": {},
-  "dependencies": {},
+  "dependencies": {
+    "gulp-shell": "^0.5.2"
+  },
```

Comment:
why do we need to do this in addition to the `gulp-shell` we have in `devDependencies`? (line 26)